app/models/organization.rb

```diff
@@ -405,13 +427,17 @@ def self.upload_invoice_to_print_vendor(file_path,file_name)
   end
 
   # Expects file_path string with file_name format /hbxid_mmddyyyy_invoices_r.pdf
+  # Also using this when uploading broker commision statements where the file_path
+  # string is a file_name with format /hbx_id_mmddyyyy_commission_NUM-NUM_R.pdf
   # Returns Organization
   def self.by_invoice_filename(file_path)
     hbx_id= File.basename(file_path).split("_")[0]
     Organization.where(hbx_id: hbx_id).first
   end
 
   # Expects file_path string with file_name format /hbxid_mmddyyyy_invoices_r.pdf
+  # Also using this when uploading broker commision statements where the file_path
+  # string is a file_name with format /hbx_id_mmddyyyy_commission_NUM-NUM_R.pdf
```

Comment:
kind of like the above comment. this comment might make it confusing in the future if changes are needed to invoices. The comment here mentions both invoice and commissions. Make a generic one or create one for commissions too.

Reply:
replaced this with a custom method for commission statements. Additionally I further decoupled the flow from invoices by adding a new method to return the organization associated with a commission statement file path